R/CONCOR_blockmodeling.R

```diff
@@ -13,32 +13,101 @@ make_blk <- function(adj_list, nsplit = 1) {
 }
 
 .edge_dens <- function(adj_mat) {
+  ## Make weighted matrix unweighted
   adj_mat[adj_mat > 0] <- 1
-  a <- sum(adj_mat)
-  m <- length(adj_mat) - sqrt(length(adj_mat))
-  d <- a / m
+
+  a <- sum(adj_mat)   # Count total edges in network
+  m <- length(adj_mat) - sqrt(length(adj_mat))  ## Count possible edges (remove the diagonal)
+  d <- a / m  # actual/possible
```

Comment:
Only change here is to add comments, right? Crazy as I think it is, those are supposed to be stripped out of submitted packages.

Reply:
I just removed the comments in my most recent push.